lib/config.yml

```diff
@@ -127,6 +127,12 @@ RSpec/VerifiedDoubles:
 Style/ClassCheck:
   Enabled: false
 
+Style/Documentation:
+  Enabled: false
+
+Style/FrozenStringLiteralComment:
```

Comment:
We'd have to add those magic comments to all our files (or rather, rubocop's autocorrect would). Also [immutable strings are still experimental](https://wyeworks.com/blog/2015/12/1/immutable-strings-in-ruby-2-dot-3) and there's a few bugs they can cause so we'd have to verify a bunch of behavior. I don't see really a benefit to doing that until Ruby 3 is actually here and we want to upgrade for perf benefits or whatever.